treeherder/model/migrations/0055_autoclassify_status.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+# Original SQL:
+#
+# BEGIN;
+# --
+# -- Add field autoclassify_status to job
+# --
+# ALTER TABLE `job` ADD COLUMN `autoclassify_status` integer DEFAULT 0 NOT NULL;
+# ALTER TABLE `job` ALTER COLUMN `autoclassify_status` DROP DEFAULT;
+# COMMIT;
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0054_duplicate_failure_classifications'),
```

Comment:
This will need rebasing on top of the new migration name once the PR in bug 1312575 is updated.
Similarly, this will need renaming from `0055_autoclassify_status.py` to the reset number sequence.